cbins13/dotTheory---FE
# FE-101: Front-End Foundations (Create HTML and CSS components)

## Related Tickets
- [FE-856](https://codeandtheory.atlassian.net/browse/FE-856)

## Description
- This pull request is created for the module FE-101, where the checklist includes three tasks:

1. List down all the Global Components on the site (modules used on all pages, like the Navigation and/or the Footer)
2. List down all the Components specific to the Detail Page only
3. Write the HTML and CSS for each Component you've identified from the 2 previous task

## Validation
- Open the components text file to see the list of global and local components from the reference site
- Open the index.html file in a browser to view the HTML components of the details page of the website
- Open the styles.css to view the CSS components
